test/integration/api_test.lua

```diff
@@ -1,22 +1,22 @@
 local fio = require('fio')
-
 local t = require('luatest')
-local g = t.group('integration_api')
 
-local log = require('log')
 
-local helper = require('test.helper.integration')
+local g = t.group('integration_api')
 
+local helper = require('test.helper.integration')
 
-local http_client = require('http.client')
-local url = '/kv'
+local server_command = 'srv_kv_storage'
 
+local storage_name = 'kv'
+local storage_path = helper.path_to_storage(storage_name)
+local path_to_key = helper.key_path_formatter(storage_path)
```

Comment:
nit: think about naming - `path_to_key` looks like a path (some `string`), but it's a function.

```lua
local key_path_formatter = helper.get_key_path_formatter(endpoint)
local path = key_path_formatter(key)
```

or simply

```lua
local path = helper.format_key_path(endpoint, key)
```

Reply:
I made common `base_kv_enpoint` from previous conversation, so it's good to make helper `key_path(key)` (anyway it's a singleton):

```lua
-- in helper/integration.lua
function helper.key_path(key)
    return join_url(helper.base_kv_endpoint, key)
end
```